google/cloud/aiplatform/v1beta1/schema/predict/instance/__init__.py

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+from google.cloud.aiplatform_helpers import add_methods_to_classes_in_package
```

Comment:
The value_converter.py module is intended to be public. It will be helpful for tabular developers who need to format their prediction instances, for example.

I'll change the name of the methods intended to be private so that they have a leading underscore in their names.

Reply:
I see - perhaps `add_methods_to_classes_in_package` should be in a private module, where as `value_converter` a public module.  in that case perhaps a nested submodule `aiplatform.helpers.value_converter` would be preferred over `aiplatform_helpers.value_converter`.

Reply:
Done.